src/Uno.Extensions.Logging/HostExtensions.cs

```diff
@@ -1,8 +1,11 @@
 ﻿namespace Uno.Extensions.Logging;
 
+/// <summary>
+/// Extensions for <see cref="IHost"/> to customize logging behavior.
+/// </summary>
 public static class HostExtensions
 {
-	public static IHost ConnectUnoLogging(this IHost host, bool enableUnoLogging = true)
+	internal static IHost ConnectUnoLogging(this IHost host, bool enableUnoLogging = true)
```

Comment:
This needs to be public to allow developer to connect uno logging to the same log output as the host
We do call this internally but only for some configurations of IHost

Reply:
@nickrandolph Reverted with 5357341bbbf17d9283e952135d90d5399199e599